ruby/test/centroid_test.rb

```diff
@@ -12,7 +12,7 @@ def json_config_with_array
   end
 
   def shared_file_path
-    'config.json'
+    File.join('..','..','config.json')
```

Comment:
Should probably roll this back, since the rakefile calls the test from the root directory instead of the test directory.